README.md

```diff
@@ -10,7 +10,7 @@
 Pycytominer is a suite of common functions used to process high dimensional readouts from high-throughput cell experiments.
 The tool is most often used for processing data through the following pipeline:
 
-![pipeline](/media/pipeline.png)
+<img height="200" src="https://github.com/cytomining/pycytominer/blob/master/media/pipeline.png?raw=true">
```

Comment:
Consider increasing the height here such that the autoscaling of the image allows the text to be read. As-is, I had a difficult time reading the text in the image. I can also understand wanting to conserve space within the rendered readme. An alternative might include adding linked text "click here for an enlarged version of this image" or something along those lines.

Reply:
Good idea! Added in next series of commits

Reply:
added in c9d302e1f2a85c2aa54fbe1345130d4ab886941c